tiup/tiup-component-dm-upgrade.md

```diff
@@ -0,0 +1,29 @@
+--
+title: tiup dm upgrade
+--
+
+# tiup dm upgrade
+
+The `tiup dm upgrade` command is used to upgrade a specified cluster to a specific version.
+
+## Syntax 
+
+```shell
+tiup dm upgrade <cluster-name> <version> [flags]
+```
+
+- `<cluster-name>` is the name of the cluster to operate on. If you forget the cluster name, check it in [tiup cluster list](/tiup/tiup-component-cluster-list.md).
```

Comment:
```suggestion
- `<cluster-name>` is the name of the cluster to be operated on. If you forget the cluster name, check it in [tiup DM cluster list](/tiup/tiup-component-dm-list.md).
```

Reply:
@lucklove please help check if a similar change is also needed in the Chinese version. Thanks!

Reply:
ditto